lib/capistrano/rails_artifact.rb

```diff
@@ -1,13 +0,0 @@
-# This file will be included by default when someone sets the scm to be :rails_artifact
-
-load File.expand_path('../tasks/rails_artifact.rake', __FILE__)
-
-# These two tasks are unnecessary for this strategy
```

Comment:
you mean... saying that you don't need to do anything but add the scm :rails_artifact?